cache/fast_lru_cache.cc

```diff
@@ -304,6 +273,7 @@ Status LRUCacheShard::InsertItem(LRUHandle* e, Cache::Handle** handle,
 Cache::Handle* LRUCacheShard::Lookup(const Slice& key, uint32_t hash) {
   LRUHandle* e = nullptr;
   {
+    PREFETCH(table_.Head(hash), 0, 1);
```

Comment:
Just wondering, does this optimization alone require a fixed size hash table? What would happen if we try to prefetch a bad address in the event of a resize?

Reply:
We believe it would be okay, since prefetching is only a hint. I guess my PR message is a bit inaccurate. It sounds like we need the table to stay fixed in order to prefetch.

The main reason we are implementing this change is for the upcoming task, which is about preallocating handles.

Reply:
Reference: https://gcc.gnu.org/onlinedocs/gcc/Other-Builtins.html#index-_005f_005fbuiltin_005fprefetch